docs/Custom-Protos.md

```diff
@@ -0,0 +1,167 @@
+# Creating custom protobuffer messages
```

Comment:
These are `protobuf` messages, not `protobuffer` messages.

Reply:
OK, I changed `protobuffer->protobuf` in several bunches.